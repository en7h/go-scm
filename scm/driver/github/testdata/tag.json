{
    "ref": "refs/tags/v0.1",
    "node_id": "MDY6Q29tbWl0MTI5NjI2OTo3ZmQxYTYwYjAxZjkxYjMxNGY1OTk1NWE0ZTRkNGU4MGQ4ZWRmMTFk",
    "url": "https://api.github.com/repos/octocat/Hello-World/git/refs/tags/v0.1",
    "object": {
        "sha": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
        "type": "commit",
        "url": "https://api.github.com/repos/octocat/Hello-World/git/commits/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"
    }
}