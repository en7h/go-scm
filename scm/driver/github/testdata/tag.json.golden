{
    "Name": "v0.1",
    "Path": "refs/tags/v0.1",
    "Sha": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"
}